### DISCUSSION TITLE
0.5.3: Delete element from Package?

### DISCUSSION THREAD

**Discussion by rcd09** [cogu/autosar#121] [General]
There is no analog to 0.4's package.delete from the looks of it.  How do I remove an element from a package in 0.5.3? 

If I need to add it, I assume its the reverse of the Package append: Make sure it is an ARElement then remove from elements list and collection_map dict.  Then create unit tests for the functionality. Yes?

**Comment by cogu** [comment]
I had no Idea that function even existed in 0.4. Yes, you can even look at what package.delete is doing in 0.4 and that is pretty much your description above.

Please don't make any changes immediately. I'm like 2-3 days away from finally releasing 0.5.4.

**Reply by rcd09** [reply]
Thanks for the reply cogu.  I had a feeling you were close to the next release so I got started updating my tool's 0.5 implementation with a whole lot of changes from the 0.4 implementation being used by application engineers here.

I will wait for 0.5.4 before considering any changes to this package.

Thanks!
